lib/OpenQA/Utils.pm

```diff
@@ -399,5 +400,19 @@ sub check_download_whitelist {
     return ();
 }
 
+sub notify_workers {
+    my $ipc = OpenQA::IPC->ipc;
```

Comment:
hmm, the function is exported into the caller namespace - so the caller has to make sure IPC is imported. But of course we can also move this into IPC - or introduce yet another class for this ;(